rskj-core/src/main/java/co/rsk/peg/BridgeStorageConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package co.rsk.peg;
+
+import org.ethereum.config.BlockchainConfig;
+
+public class BridgeStorageConfiguration {
+    private Boolean isUnlimitedWhitelistEnabled;
```

Comment:
Can we use `boolean` instead of `Boolean`? It's preferred to use primitive types over boxed values.